docs/develop/api-reference/set-collections.md

```diff
@@ -164,3 +164,23 @@ The response object for SetContainsElements returns three possible options, a ca
 See [response objects](./response-objects.md) for specific information.
 
 </details>
+
+### SetLength
+Get the length of an existing set item
+
+| Name      | Type         | Description                         |
+|-----------| ------------ |-------------------------------------|
+| cacheName | String       | Name of the cache.                  |
+| setName   | String       | Name of the set item to be checked. |
```

Comment:
Are these key properties strings or strings/byte array? Are all our commands the same? I've seen both in this PR